_layouts/quality_estimation.html

```diff
@@ -12,19 +12,17 @@ <h2>
 <hr>
 
 <p>
-  {% if page.supported_languages.size > 0 %}
-    The <strong>{{ page.title }}</strong> quality estimation API supports <a href="#language-support">{{ page.supported_languages | size }} languages</a>.
-  {% else %}
-    The <strong>{{ page.title }}</strong> quality estimation API supports an unknown number of languages.
-  {% endif %}
+  The <strong>{{ page.title }}</strong> quality estimation API is available in
+  <a href="#integrations">{{ page.integrations | size }}
+  machine translation management system{% if page.integrations.size == 1 %}s{% endif %}</a>.
```

Comment:
Can we have different versions depending on the QE API integrations count? Since there aren't a lot of integrations yet, only 0, 1 or 2, I think singular and plural sentences are too visible.

![Screenshot 2023-09-26 at 11 04 28](https://github.com/machinetranslate/machinetranslate.org/assets/33723604/a0fbd6a1-ef4b-4588-b92d-395a1ff975fd)

Reply:
I fixed this, should be `!= 1` not `== 1`.

Once we had connectors etc that already exist (a bit like aggregators for MT), it will change to 5 or so for ModelFront and I can only assume that the others will launch more too.  I just didn't want to do that in this PR.

There will be a few like Phrase MTQE or KantanQES that are only available in 1 and for that I think it is useful to be explicit, it is a bit like the Crowdin MT, i.e. that property of being very tied to that TMS is very important.

I'm also trying to keep us from having too much repeated code for plural forms, both here and in apis.html etc.  I know I added it originally, but I regret it - more code, hard to maintain, all for questionable benefit to the user.